### PR TITLE
Fix scenario failures on K8S due to missing host information

### LINKED ISSUE #561: Scenarios fail on K8S due to missing host information in shell deployer
**Description:**
Scenarios are failing on K8S on OpenStack due to missing host information. Tomcat host information and deployment hosts' information should be added into ShellDeployer

**Suggested Labels:**
Type/Bug

**Suggested Assignees:**
AsmaJ

**Steps to reproduce:**
Run K8S on OpenStack on TestGrid

### PR DESCRIPTION
**Pull request by azinneera**
**Purpose**
Contains changes related to adding the necessary host information in ShellDeployer.
Resolves  #561

**Goals**
Fixing the test failures occurring due to bugs in TestGrid

**Approach**
Hard-coded tomcat host information
Retrieved other host information of the deployment through deployment.json generated by deploy.sh

**Security checks**
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java [hunk 1]
```diff
@@ -89,7 +93,21 @@ public DeploymentCreationResult deploy(TestPlan testPlan,
...
+        tomcatHost.setLabel("tomcatHost");
+        tomcatHost.setIp("ec2-52-54-230-106.compute-1.amazonaws.com");
+        Host tomcatPort = new Host();
+        tomcatPort.setLabel("tomcatPort");
+        tomcatPort.setIp("8080");
+        hosts.add(tomcatHost);
+        hosts.add(tomcatPort);
+
+        DeploymentCreationResult result1 = DeploymentUtil.getDeploymentCreationResult(infrastructureProvisionResult
+                .getDeploymentScriptsDir());
```
Comment (kasunbg) [inline deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java]: What's the reason to create another DeploymentCreationResult instance? Can't we use `result` instance?

Comment (azinneera) [reply deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java]: Yes, we can. Made changes in e8bb16e

File: deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java [hunk 2]
```diff
@@ -80,16 +83,32 @@ public DeploymentCreationResult deploy(TestPlan testPlan,
...
+
+        List<Host> hosts = new ArrayList<>();
+        Host tomcatHost = new Host();
+        tomcatHost.setLabel("tomcatHost");
+        tomcatHost.setIp("ec2-52-54-230-106.compute-1.amazonaws.com");
+        Host tomcatPort = new Host();
+        tomcatPort.setLabel("tomcatPort");
+        tomcatPort.setIp("8080");
+        hosts.add(tomcatHost);
+        hosts.add(tomcatPort);
```
Comment (harshanL) [inline deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java]: Please create another issue for obtaining tomcat server details from either a configuration or database.

Comment (azinneera) [reply deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java]: Created #567

**Pull request merged**